src/sshagent/OpenSSHKey.cpp

```diff
@@ -135,6 +196,11 @@ void OpenSSHKey::setType(const QString& type)
     m_type = type;
 }
 
+void OpenSSHKey::setCheck(quint32 check)
+{
+    m_check = check;
+}
```

Comment:
This function name and purpose is obtuse. What is meant by check? What happens if I don't call this function? We don't do any checks in the privateKey() function to ensure the value is valid.

Reply:
It's just a random integer that is tested to check the integrity of a decrypted key. I just used the same variable name as OpenSSH developers :sweat_smile: 

https://github.com/openssh/openssh-portable/blob/master/sshkey.c#L3958